### PR TITLE
fabtests/pytest/conftest.py: increate connect timeout

### PR DESCRIPTION
**Pull request by silany**
Increase connect timeout

Signed-off-by: silany <silany@amazon.com>

### CODE REVIEW

File: fabtests/pytest/conftest.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def pytest_addoption(parser):
                              help=option_helpmsg, default=option_default)
 
 # base ssh command
-bssh = "ssh -n -o StrictHostKeyChecking=no -o ConnectTimeout=30 -o BatchMode=yes"
+bssh = "ssh -n -o StrictHostKeyChecking=no -o ConnectTimeout=120 -o BatchMode=yes"
```
Comment (wenduwan) [inline fabtests/pytest/conftest.py:41]: Thanks for the PR. Do you mind explaining how come 30 seconds SSH connection timeout is not sufficient? It is unusual for the connection to take more than that.

Comment (silany) [reply fabtests/pytest/conftest.py:41]: Hey, we are currently using 120 in other benchmarks/apps, but getting connection timeout when testing fabtests in a specific scenario
I would like to try and see if it solves timeouts. 
Not sure yet, this should be merged

Comment (wenduwan) [reply fabtests/pytest/conftest.py:41]: Were you able to fully mitigate the timeout with 120 seconds in your local test setup? Some verbose logging can be very helpful, e.g. `ssh -vvv`.

What particular test was impacted by this, e.g. was `ssh ConnectTimeout=30 localhost` ever timed out?

**Comment by wenduwan** [comment]
Typo in commit message, i.e. `increase`

**Comment by wenduwan** [comment]
Also there is a test [timeout parameter](https://github.com/wenduwan/libfabric/blob/bec93b185b5448154348c26fdff100caba9d38c3/fabtests/pytest/options.yaml#L70) which actually limits how long the `fi_*` command runs. Throwing in here to make sure we are changing the right param.

**Comment by wenduwan** [comment]
@silany are you still affected by this issue?

**Comment by wzamazon** [comment]
@silany Do you still need this PR?

**Comment by wenduwan** [comment]
No response from requester. Closing for now. Please reopen if issue persists.

**Pull request closed without merging**